### PR TITLE
 finished vanilla policy gradient baseline implementation on four rooms

### PR DESCRIPTION
**Pull request by bohanwu**

### CODE REVIEW

File: test_env/envs/fourrooms.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def inc(row, col, a):
                         newstate = to_s(newrow, newcol)
                         newletter = desc[newrow, newcol]
                         done = bytes(newletter) in b'GH'
-                        rew = float(newletter == b'G')
+                        rew = float(newletter == b'G') * 100 - 1
```
Comment (ByzanTine) [inline test_env/envs/fourrooms.py]: What's the purpose of *100?

Comment (bohanwu) [reply test_env/envs/fourrooms.py]: Just so that the reward is more readable

File: config.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    batch_size_by_env = {
+        "Fourrooms-v1": 1000
+    }
+
+    lr_by_env = {
+        "Fourrooms-v1": 3e-2
+    }
+
+    gamma_by_env = {
+        "Fourrooms-v1": 1.0
```
Comment (ByzanTine) [inline config.py]: I think gamma should be less than 1 in four room case, otherwise any policy is optimal...

File: config.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    }
+    max_epsilon = 0
+    min_epsilon = 0
```
Comment (ByzanTine) [inline config.py]: Should epsilons be at least >0 for test purpose?

File: config.py (hunk 2 above)
Comment (bohanwu) [reply config.py]: Oh so I did a -1 on each time step, didn’t work

File: config.py (hunk 3 above)
Comment (bohanwu) [reply config.py]: Yeah epsilon was positive, I tried that too. Didn’t work

Comment (bohanwu) [reply config.py]: The code is here, and we might be able to make it learn if we tune parameters diligently enough. I will leave this decision to you guys. I was thinking that the baseline should just be the baseline under no significant tuning

Comment (ByzanTine) [reply config.py]: I won't be convinced that policy gradient, in this case Actor critic doesn't work on 4-rooms with tabular states. I will run my experiment (based on your stuff) soon, let's wait for a bit on this item.

Comment (bohanwu) [reply config.py]: Sounds good. It could work. Let’s see it.

**Pull request closed without merging**